darts/models/forecasting/forecasting_model.py

```diff
@@ -1773,27 +1838,49 @@ def _evaluate_combination(param_combination) -> float:
                     last_points_only=last_points_only,
                     verbose=verbose,
                     show_warnings=show_warnings,
+                    data_transformers=data_transformers,
                     fit_kwargs=fit_kwargs,
                     predict_kwargs=predict_kwargs,
                     sample_weight=sample_weight,
                 )
             else:  # split mode
+                if len(data_transformers) > 0:
```

Comment:
should we also apply this to the `use_fitted_values=True` case?

Reply:
`fitted_values` is actually supported only by very models, I am not sure that it's relevant to add data transformer here... But going to do it, just in case